projet/projects/views.py

```diff
@@ -53,12 +53,40 @@ def get_queryset(self):
         return Project.objects.filter(user=self.request.user)
 
 
-class ProjectDetailUpdateDelete(generics.RetrieveUpdateDestroyAPIView):
```

Comment:
Had to change to a generic view because of certain intricate user cases. Like the user either has to have the project he is getting the detail of in his contributions relations or has to be the manager of the project.